sql/src/main/java/io/crate/expression/symbol/format/SymbolPrinter.java

```diff
@@ -274,6 +392,20 @@ private void printArgs(List<Symbol> args, SymbolPrinterContext context) {
                 }
             }
         }
+
+        private static boolean isTableFunctionReference(Reference reference) {
+            RelationName relationName = reference.ident().tableIdent();
+            return relationName.equals(UnnestFunction.TABLE_IDENT) || relationName.equals(EmptyRowTableFunction.TABLE_IDENT);
```

Comment:
This is bound to break as soon as we add other table functions. I think we need to come up with a different way to detect if a table function is involved. Or change the structure somehow.